src/Bigcommerce/Api/Connection.php

```diff
@@ -97,6 +97,7 @@ public function __construct()
         $this->curl = curl_init();
         curl_setopt($this->curl, CURLOPT_HEADERFUNCTION, [$this, 'parseHeader']);
         curl_setopt($this->curl, CURLOPT_WRITEFUNCTION, [$this, 'parseBody']);
+        curl_setopt($this->curl, CURLOPT_USERAGENT, 'PHP Curl');
```

Comment:
Can we use something more descriptive here? Maybe `PHP CURL - Bigcommerce API Client`?

Reply:
definitely, I was struggling to come up with a name

Reply:
done